src/Toolbar.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import 'semantic-ui-css/semantic.min.css';
```

Comment:
dont need this. the css can be loaded like this in index.html:
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.11/semantic.min.css"></link>